architecture/understanding-development.adoc

```diff
@@ -173,7 +173,7 @@ Red Hat Registry. The Red Hat Registry is represented by two locations:
 `registry.access.redhat.com`, which is unauthenticated and deprecated, and
 `registry.redhat.io`, which requires authentication. You can learn about the Red
 Hat and partner images in the Red Hat Registry from the
-link:https://registry.redhat.io/[Red Hat Container Catalog].
+link:https://catalog.redhat.com/software/containers/explore[Container Images section of the Red Hat Ecosystem Catalog].
```

Comment:
If we do reopen this PR for inclusion, "Container Images" should be "Container images" throughout the R.

Reply:
Hi! Yes, I'll reopen this PR. Thank you for that note.

Reply:
Thanks a lot @bmcelvee!